ajax/libs/rpg-awesome

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "rpg-awesome",
+  "filename": "rpg-awesome.min.css",
```

Comment:
Let's modify to `css/rpg-awesome.min.css`
Because the file structure made the file under css folder.
Thanks